python_modules/dagster/dagster/_core/execution/plan/execute_step.py

```diff
@@ -131,7 +131,7 @@ def _process_user_event(
         output_name = step_context.job_def.asset_layer.get_output_name_for_asset_check(
             asset_check_evaluation.asset_check_handle
         )
-        output = Output(value=None, output_name=output_name)
+        output = Output(value=asset_check_evaluation, output_name=output_name)
```

Comment:
Can fetch the check evaluations through the event storage instead, but I turned on io managers for them here. Makes the io manager case nicer

Reply:
Yeah this is nice.

Reply:
replaced with fetching via checks table for now, since that works with both the io managed and not managed cases